src/mongoc/mongoc-socket.c

```diff
@@ -487,7 +487,7 @@ mongoc_socket_accept_ex (mongoc_socket_t *sock,      /* IN */
 int
 mongoc_socket_bind (mongoc_socket_t       *sock,    /* IN */
                     const struct sockaddr *addr,    /* IN */
-                    socklen_t              addrlen) /* IN */
+                    mongoc_socklen_t              addrlen) /* IN */
```

Comment:
here, and everywhere else, the argument should be aligned in a pretty ascii art with the argument above